target/linux/ramips/dts/mt7620a_netgear_ex6120.dts

```diff
@@ -0,0 +1,58 @@
+/* This file is released into the public domain */
```

Comment:
Please add an SPDX license identifier.

Reply:
There's two contributors to target/linux/ramips/dts/mt7620a_netgear_ex6130.dts  - @adrianschmutzler and @playya 
target/linux/ramips/dts/mt7620a_netgear_ex6120.dts is a copy of that file - if 6120 lacks something, please ask those people to amend the original file as well. I'd give it a CC-PDDC if I was the original author.

Reply:
I've asked myself and I think you should add a SPDX license identifier to this file.

Despite, the license is for the content, the words are "trivial" here, so you pick the license for the leds, keys, partitions etc. that you have verified. If you just copied the GPIO numbers without verifying them, so the other files license would be relevant, then we should close this anyway ...